### PR TITLE
Fetch time till hardLimit clears

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Hard limit reached date is hard coded](https://trello.com/c/Zg4fSQIY/139-hard-limit-reached-date-is-hard-coded)

File: lib/customers.js [hunk 1]
```diff
@@ -115,17 +117,32 @@ function getById (id, userToken) {
...
+  let startDate = moment()
+  startDate = startDate.subtract(1, 'days')
+
+  const cashInMoment = moment(cashInDate || startDate)
+  const cashOutMoment = moment(cashOutDate || startDate)
+
+  const cashInDuration = moment.duration(cashInMoment.diff(startDate)).asHours()
+  const cashOutDuration = moment.duration(cashOutMoment.diff(startDate)).asHours()
+
+  return _.ceil(_.max([cashInDuration, cashOutDuration]))
```
Comment (joshmh) [inline lib/customers.js]: Do we need to check for negative values?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/customers.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/customers.js]: Not really because of the sql 'where'.

Comment (joshmh) [reply lib/customers.js]: Hmm, maybe but I think an extra check in the logic here will make it safer.

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/customers.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/customers.js]: You're absolutely right =)

**Pull request merged**